### PR TITLE
allow override sleep_duration_secs

### PR DESCRIPTION
**Pull request by BumbleFeng**
relate to https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/20844

Signed-off-by: Feng Huang <fehuang@redhat.com>

### CODE REVIEW

File: openshift/qontract-reconcile-internal.yaml [hunk 1]
```diff
@@ -7172,6 +7172,8 @@ parameters:
   value: 100m
 - name: GITLAB_OWNERS_MEMORY_REQUEST
   value: 100Mi
+  name: GITLAB_HOUSEKEEPING_SLEEP_DURATION_SECS
+  value: "0"
```
Comment (jfchevrette) [inline openshift/qontract-reconcile-internal.yaml:7176]: Should we have default that isn't zero?

File: helm/qontract-reconcile/values-internal.yaml [hunk 2]
```diff
@@ -204,6 +204,7 @@ integrations:
       memory: 200Mi
       cpu: 200m
 - name: gitlab-housekeeping
+  sleep_duration_secs: "0"
```
Comment (jfchevrette) [inline helm/qontract-reconcile/values-internal.yaml:207]: This look like this will run continually (no sleep). Is this desired?

Comment (BumbleFeng) [reply helm/qontract-reconcile/values-internal.yaml]: GitLab housekeeping can only merge one MR for app-interface in one loop. The faster it iterates, the more quickly MR can be merged. In the case of foolproof, I have changed the default value in template to 600 and override the parameter in the saas file.

**Review by jfchevrette — APPROVED** [review]

File: helm/qontract-reconcile/values-internal.yaml [hunk 3]
```diff
@@ -204,6 +204,7 @@ integrations:
       memory: 200Mi
       cpu: 200m
 - name: gitlab-housekeeping
+  sleep_duration_secs: 600
```
Comment (maorfr) [inline helm/qontract-reconcile/values-internal.yaml:207]: in helm we should use `sleepDurationSecs`

**Review by maorfr — APPROVED** [review]
LGTM

**Pull request merged**